pkg/plugin/resources.go

```diff
@@ -175,6 +175,12 @@ func (app *App) handleReport(w http.ResponseWriter, req *http.Request) {
 			HeaderName:  backend.CookiesHeaderName,
 			HeaderValue: req.Header.Get(backend.CookiesHeaderName),
 		}
+	// Requires Grafana v10.3.0+ and feature flag "idForwarding" enabled
+	case req.Header.Get("X-Grafana-Id") != "":
```

Comment:
Maybe defining a constant for `X-Grafana-Id` is worth as we are using it at least 3 times?